### PR TITLE
Add documentation for the new cluster-health alerts.

### PR DESCRIPTION
**Pull request by YoEight**

### CODE REVIEW

File: docs/cloud/integrations/README.md [hunk 1]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
...
 The issue closes when the gossip status again returns expected values.
 
+### Elections
+
+For a cluster, an issue is opened if there is more than one leader change per hour. The issue closes only if there is no more
+than one leader change in the hour following the first detection.
+
+### Leader Mismatches
+
+For a cluster, an issue is opened if the cluster disagrees on which node is the current leader in a three minutes time window.
```
Comment (TimSimpson) [inline docs/cloud/integrations/README.md]:
```suggestion
For a cluster, an issue is opened if the cluster disagrees on which node is the current leader in a three minute time window.
```

File: docs/cloud/integrations/README.md [hunk 2]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
...
+The issue closes if the cluster agrees on which node is the leader one minute after the first detection.
+
+### Out of Syncs
+
+For a cluster, an issue is opened if at least one follower node hasn't caught up with the leader node progression from two seconds ago, over a three minutes time window. The issue closes if all the followers are keeping up with the leader node progression over a three minutes time window, after the first detection.
```
Comment (TimSimpson) [inline docs/cloud/integrations/README.md]:
```suggestion
For a cluster, an issue is opened if at least one follower node hasn't caught up with the leader node progression from two seconds ago, over a three minute time window. The issue closes if all the followers are keeping up with the leader node progression over a three minute time window, after the first detection.
```

File: docs/cloud/integrations/README.md [hunk 3]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
...
+
+### Unreachable Nodes
+
+For each node of the cluster, if one node is not reachable from another one over a three minutes time window. The issue closes
```
Comment (TimSimpson) [inline docs/cloud/integrations/README.md]:
```suggestion
For each node of the cluster, if one node is not reachable from another one over a three minute time window. The issue closes
```

**Review by TimSimpson — APPROVED** [review]
This does a good job of summarizing the new issues. `minutes` should be `minute` in a few spots, but beyond that looks good!

**Review by ylorph — COMMENTED** [review]
Does any of those issue have a 
"If this happens, consider ...." ?

File: docs/cloud/integrations/README.md (hunk 1 above)
Comment (ylorph) [reply docs/cloud/integrations/README.md]: Can a cluster "disagree" ?  
I mean, is it standard terminology in consensus algorithm ?  

> Fires when nodes in a cluster report different leaders in a three minute time window
> The issues closes automatically if all the nodes reports the same leader in the minute after detection

File: docs/cloud/integrations/README.md [hunk 4]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
...
 
+### Elections
+
+For a cluster, an issue is opened if there is more than one leader change per hour. The issue closes only if there is no more
+than one leader change in the hour following the first detection.
+
+### Leader Mismatches
+
+For a cluster, an issue is opened if the cluster disagrees on which node is the current leader in a three minutes time window.
+The issue closes if the cluster agrees on which node is the leader one minute after the first detection.
```
Comment (ylorph) [inline docs/cloud/integrations/README.md:59]: What happens if there are still multiple leaders detected after the 1 minute ?

Any action the users can take ?

File: docs/cloud/integrations/README.md [hunk 5]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
...
+
+### Out of Syncs
+
+For a cluster, an issue is opened if at least one follower node hasn't caught up with the leader node progression from two seconds ago, over a three minutes time window. The issue closes if all the followers are keeping up with the leader node progression over a three minutes time window, after the first detection.
+
+### Unreachable Nodes
+
+For each node of the cluster, if one node is not reachable from another one over a three minutes time window. The issue closes
+if all nodes are reachables after a three minutes time window, after the first detection.
```
Comment (ylorph) [inline docs/cloud/integrations/README.md:68]: "The issue closes automatically if all nodes can reach each other after a three minute time window beginging at the first detection"


- What happens after the 3 minute window if the still can't reach each other ?
- Any actions the users can take ?

File: docs/cloud/integrations/README.md [hunk 6]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
 
 The issue closes when the gossip status again returns expected values.
 
+### Elections
+
+For a cluster, an issue is opened if there is more than one leader change per hour. The issue closes only if there is no more
+than one leader change in the hour following the first detection.
+
+### Leader Mismatches
```
Comment (alexeyzimarev) [inline docs/cloud/integrations/README.md]:
```suggestion
### Leader mismatches
```

File: docs/cloud/integrations/README.md [hunk 7]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
...
+
+For a cluster, an issue is opened if the cluster disagrees on which node is the current leader in a three minute time window.
+The issue closes if the cluster agrees on which node is the leader one minute after the first detection.
+
+### Out of Syncs
```
Comment (alexeyzimarev) [inline docs/cloud/integrations/README.md]:
```suggestion
### Node out of syncs
```

File: docs/cloud/integrations/README.md [hunk 8]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
 
 The issue closes when the gossip status again returns expected values.
 
+### Elections
```
Comment (alexeyzimarev) [inline docs/cloud/integrations/README.md]:
```suggestion
### Frequent elections
```

File: docs/cloud/integrations/README.md [hunk 9]
```diff
@@ -48,6 +48,25 @@ Every node on a cluster has it's gossip status queried twice each minute. An iss
...
+### Leader Mismatches
+
+For a cluster, an issue is opened if the cluster disagrees on which node is the current leader in a three minute time window.
+The issue closes if the cluster agrees on which node is the leader one minute after the first detection.
+
+### Out of Syncs
+
+For a cluster, an issue is opened if at least one follower node hasn't caught up with the leader node progression from two seconds ago, over a three minute time window. The issue closes if all the followers are keeping up with the leader node progression over a three minute time window, after the first detection.
+
+### Unreachable Nodes
```
Comment (alexeyzimarev) [inline docs/cloud/integrations/README.md]:
```suggestion
### Unreachable nodes
```

**Review by alexeyzimarev — CHANGES_REQUESTED** [review]

**Review by alexeyzimarev — APPROVED** [review]

File: docs/cloud/integrations/README.md (hunk 2 above)
Comment (ylorph) [inline docs/cloud/integrations/README.md]: Not sure I totally understand the "2 seconds ago" .

    What happens when the node are not caught up after the 2 minute period.

    Any action the users can take ?

File: docs/cloud/integrations/README.md (hunk 5 above)
Comment (alexeyzimarev) [reply docs/cloud/integrations/README.md:68]: Maybe we should add "if the issue doesn't close after 15 minutes, or you observe performance degradation, limited or no availability of the cluster, contact support"

**Pull request merged**